en/features/local_census.md

```diff
@@ -1,30 +1,34 @@
-##Local Census
+# Local Census
 
 To provide to administrator users a way to manage the local census database through the administration panel **Settings > Manage local census**. Currently the only way to manipulate this table records is through the rails console.
 
 Allow administrators users to manage this table in two different ways:
 
-  - **Manually**: one by one through a CRUD interface.
-  - **Automatically**: through an importation process.
+- **Manually**: one by one through a CRUD interface.
+- **Automatically**: through an importation process.
+
+## Manually
 
-#### Manually
 Provide a way to manage local census records to administrator users through administration interface.
 
 - Local Census Page
-![Manage local census](../../img/local_census/manage-local-census-en.png)
+  ![Manage local census](../../img/local_census/manage-local-census-en.png)
+
 - Add new record
-![Create local census record](../../img/local_census/add-local-census-record-en.png)
+  ![Create local census record](../../img/local_census/add-local-census-record-en.png)
 
 Features:
 
 1. Search by document_number: As local_census_records could contain a lot of records we have added a search feature to allow administrators to find existing records by document_number.
+
```

Comment:
Why are we adding a blank line between these two items? :thinking:

Reply:
I seem to remember adding it to make it look better.
What is clear is that it is not a change that has to go in this commit, so I remove this extra line.